src/datasources/system/types.ts

```diff
@@ -1,6 +1,15 @@
 import { DataQuery } from '@grafana/schema'
 
+export enum QueryType {
+  Metadata = "Metadata",
```

Comment:
Do we actually need to reassign the default enum values here?

Reply:
I think Carson just said it was cleaner that way

Reply:
If we don't explicitly set a value here, then the query type would just get saved as integers.